package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "dao-frontend",
-  "version": "1.2.0",
+  "version": "1.3.0",
```

Comment:
Shouldn't we change the package-lock.json also?
I normally use `npm version <> --no-git-tag-version` to take care of it

Reply:
As you wish, see commit below.